recipes-devtools/python/python3-jepture_git.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "A simple python library to quickly capture images on the Jetson."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=4f0a72805ccc91536540a1cf89a935ec"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+PKG_VER = "${L4T_VERSION}${@l4t_bsp_debian_version_suffix(d)}"
+SRC_SOC_DEBS = "nvidia-l4t-jetson-multimedia-api_${PKG_VER}_arm64.deb;subdir=git/jetson_multimedia_api"
+
+inherit setuptools3 l4t_deb_pkgfeed
+
+SRC_URI =+ " \
+    git://github.com/DelSkayn/jepture.git;protocol=https;branch=main \
+    file://0001-setup.py-don-t-hardcode-paths.patch \
+    file://0002-v4l2-remove-duplicate-symbols-from-nvidia-extensions.patch \
+    file://0003-Revert-argus_stream-update-certain-symbols-to-work-w.patch \
+"
+
+python __anonymous() {
+    chksums = {
+        "tegra186": "40225cf5279fc8ded8a29966e7923d1bdf5a7eb9ff5bee2c016fec2db40a786a",
+        "tegra194": "40225cf5279fc8ded8a29966e7923d1bdf5a7eb9ff5bee2c016fec2db40a786a",
+        "tegra210": "141c7046d7eee5f39c216ab12f758577257689678765fe54972cfeff91bd4db2",
+    }
```

Comment:
You can just use the t186 deb package; the content is identical in the t210 one.  Doing so will eliminate rebuilds between platforms and will let you drop the anonymous python rigmarole.  See the existing tegra-mmapi recipes in meta-tegra.

Reply:
Done